go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -877,3 +899,31 @@ func (bsnc *BaseSecondaryNetworkController) requireDHCP(pod *corev1.Pod) bool {
 		bsnc.IsPrimaryNetwork() &&
 		bsnc.TopologyType() == types.Layer2Topology
 }
+
+func (bsnc *BaseSecondaryNetworkController) createVMSourcePodLSPOpsForLiveMigration(
```

Comment:
The fact that they are ops, is usually on ovn-k notated with `Ops` suffix, so let's rename this with what it's really doing `disableLiveMigrationSourceLSPOps`

Reply:
ACK, DONE